controllers/changepackage.lua

```diff
@@ -73,15 +73,23 @@ return {
     assert_error(self.session.name, self.i18n("err_not_logged_in"))
     assert_error(self.session.verified, self.i18n("err_email_not_verified"))
     local user = Users:get_user(self.session.name)
-    self.myPackages = Packages:select("* where user_id = "..user.id)
+    if not user.admin then
+      self.myPackages = Packages:select("* where user_id = "..user.id)
```

Comment:
Don't concatenate values like this - it's how SQL injections happen :wink: check out https://leafo.net/lapis/reference/database.html#making-a-query on how to do them instead.

Reply:
I put it into models and it's not a concatenation anymore :)

Reply:
Very nice :)